site/src/pages/LoginPage/SignInForm.tsx

```diff
@@ -83,6 +85,10 @@ export const SignInForm: FC<SignInFormProps> = ({
   const passwordEnabled = authMethods?.password.enabled ?? true;
   const applicationName = getApplicationName();
 
+  const [tosAccepted, setTosAccepted] = useState(false);
```

Comment:
I was expecting this to be filled by some query data.

Reply:
we're adding this for a single customer, and Kyle doesn't want any more backend than is absolutely necessary.